v2/tools/generator/internal/config/version_configuration.go

```diff
@@ -56,6 +56,16 @@ func (vc *VersionConfiguration) TypeRename(name string) (string, error) {
 	return rename, nil
 }
 
+// FindUnusedTypeRenames returns a slice listing any unused type rename configuration
+func (vc *VersionConfiguration) FindUnusedTypeRenames() []string {
```

Comment:
I don't think there's so much point in doing what I suggested for the group configuration here.

Reply:
Still two places to use the helper function, and that number will increase. So, I made the change. 😁